bots/task/testmap.py

```diff
@@ -0,0 +1,123 @@
+#!/usr/bin/env python3
```

Comment:
This shouldn't have a shebang at all (nor be executable), this is just a module. There's no `__main__`.